index.ts

```diff
@@ -33,18 +32,21 @@ app.use(
 );
 app.use('/api', api);
 app.use('/api', upload);
-app.use(
-  '/graphql',
-  graphqlHTTP({
-    schema,
-    rootValue,
-    graphiql: { defaultQuery },
-    validationRules: [queryCountLimit(5, 5)]
-  })
-);
-app.get('/*', (req, res) => res.redirect('/api'));
 
 const PORT = process.env.PORT || 3000;
-app.listen(PORT, () =>
-  console.log(`Snapshot hub started on: http://localhost:${PORT}`)
-);
+const server = new ApolloServer({
+  schema,
+  rootValue,
+  playground: true,
```

Comment:
I think this is the default value we can remove it, i got the playground without it using another repo.

Reply:
@bonustrack you are able to get this playground on heroku?? 🤔 we need it on production also right? 
because in Heroku `NODE_ENV` will be defaulted to `production`

```
When NODE_ENV is set to production, GraphQL Playground (as well as introspection) is disabled as a production best-practice.
```
https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/

Reply:
we need `playground: true,` for production to get it running

Reply:
Ho yes good catch